changelog/_template.rst

```diff
@@ -0,0 +1,40 @@
+{% for section in sections %}
+{% set underline = "-" %}
+{% if section %}
+{{section}}
+{{ underline * section|length }}{% set underline = "~" %}
+
+{% endif %}
+{% if sections[section] %}
+{% for category, val in definitions.items() if category in sections[section] %}
+
+{{ definitions[category]['name'] }}
+{{ underline * definitions[category]['name']|length }}
+
+{% if definitions[category]['showcontent'] %}
+{% for text, values in sections[section][category]|dictsort(by='value') %}
+{% set issue_joiner = joiner(', ') %}
+- {% for value in values|sort %}{{ issue_joiner() }}`{{ value }} <https://github.com/ansible/molecule/issues/{{ value[1:] }}>`_{% endfor %}: {{ text }}
```

Comment:
I think, this should use extlinks declared in the sphinx config — https://github.com/ansible/molecule/blob/6b6b2bd/doc/source/conf.py#L72-L75
This is how:
```suggestion
- {% for value in values|sort %}{{ issue_joiner() }}:issue:`{{ value }} <{{ value[1:] }}`_{% endfor %}: {{ text }}
```
There's: `:issue:`, `:pr:`, `:commit:` and `:gh:` defined.
`:gh:` one is already used in the changelog, for example: https://github.com/ansible/molecule/blame/master/CHANGELOG.rst#L81